test/commons/math/get-offset.js

```diff
@@ -1,104 +1,66 @@
-    assert.closeTo(getOffset(nodeB, nodeA), 30, round);
+  it('returns center to edge of circle when both are undersized', () => {
+    const fixture = fixtureSetup(`
+      <button style="width: 10px; height: 10px; margin: 0; position: absolute; top: 0; left: 0">&nbsp;</button>
+      <button style="width: 10px; height: 10px; margin: 0; position: absolute; top: 50px; left: 0">&nbsp;</button>
+    `);
+    const nodeA = fixture.children[1];
+    const nodeB = fixture.children[3];
+    assert.closeTo(getOffset(nodeA, nodeB), 38, round);
   });
 
-  it('returns height + spacing for vertically adjacent elms', function () {
-    var fixture = fixtureSetup(
-      '<a style="height:30px; margin:10px 0; display:block">&nbsp;</a>' +
-        '<b style="height:20px; display:block">&nbsp;</b>'
-    );
-    var nodeA = fixture.children[0];
-    var nodeB = fixture.children[1];
-    assert.closeTo(getOffset(nodeA, nodeB), 40, round);
-    assert.closeTo(getOffset(nodeB, nodeA), 30, round);
-  });
-
-  it('returns closest vertical distance for elements horizontally aligned', function () {
-    var fixture = fixtureSetup(
-      '<a style="height:30px; margin:10px 0; display:block">&nbsp;</a>' +
-        '<b style="height:20px; margin: 0 10px; display:block">&nbsp;</b>'
-    );
-    var nodeA = fixture.children[0];
-    var nodeB = fixture.children[1];
-
-    assert.closeTo(getOffset(nodeA, nodeB), getDiagonal(40, 10), round);
-    assert.closeTo(getOffset(nodeB, nodeA), 30, round);
+  it('returns center to edge of square when one is undersized', () => {
+    const fixture = fixtureSetup(`
+      <button style="width: 10px; height: 10px; margin: 0; position: absolute; top: 0; left: 0">&nbsp;</button>
+      <button style="width: 30px; height: 30px; margin: 0; position: absolute; top: 50px; left: 0">&nbsp;</button>
+    `);
+    const nodeA = fixture.children[1];
+    const nodeB = fixture.children[3];
+    assert.closeTo(getOffset(nodeA, nodeB), 45, round);
   });
 
-  it('returns corner to corner distance for diagonal elms', function () {
-    var fixture = fixtureSetup(
-      '<a style="width: 30px; height:30px; margin:10px 30px; display:block">&nbsp;</a>' +
-        '<b style="width: 20px; height:20px; display:block">&nbsp;</b>'
-    );
-    var nodeA = fixture.children[0];
-    var nodeB = fixture.children[1];
-    assert.closeTo(getOffset(nodeA, nodeB), getDiagonal(40), round);
-    assert.closeTo(getOffset(nodeB, nodeA), getDiagonal(30), round);
+  it('returns center to corner or square when at a diagonal', () => {
+    const fixture = fixtureSetup(`
+      <button style="width: 10px; height: 10px; margin: 0; position: absolute; top: 0; left: 0">&nbsp;</button>
+      <button style="width: 30px; height: 30px; margin: 0; position: absolute; top: 50px; left: 50px">&nbsp;</button>
+    `);
+    const nodeA = fixture.children[1];
+    const nodeB = fixture.children[3];
+    assert.closeTo(getOffset(nodeA, nodeB), 61.5, round);
```

Comment:
Is that right? You're going from x:5,y:5 to x:50,y:50. `Math.hypot(45, 45)` is 63.63.... Why is this 61.5?

Reply:
Padding apparently. Good catch.